kratos/includes/key_hash.h

```diff
@@ -341,6 +341,41 @@ namespace Kratos
         }
     };
 
+    /**
+     * @brief This is a hasher for index pairs
+     * @details Used for example for edges map
+     */
+    struct IndexPairHasher
+    {
```

Comment:
There is no hash function for pair in std?

Reply:
I didn't check since this was an `<int,int>` to `bool` map.

Reply:
Apparently not

Reply:
AFAIK `std::hash` is specialized for `std::pair`

Reply:
I will check the change (the original code is not mine)

Reply:
I changed for a general std::pair hash, but std::hash does not work for a std::pair directly, I tried and does not compile